lib/internal/process/next_tick.js

```diff
@@ -149,24 +147,6 @@ function setupNextTick() {
     }
   }
 
-  // TODO(trevnorris): Using std::stack of Environment::AsyncHooks::ids_stack_
-  // is much slower here than was the Float64Array stack used in a previous
-  // implementation. Problem is the Float64Array stack was a bit brittle.
-  // Investigate how to harden that implementation and possibly reintroduce it.
-  function nextTickEmitBefore(asyncId, triggerAsyncId) {
-    if (async_hook_fields[kBefore] > 0)
-      emitBefore(asyncId, triggerAsyncId);
-    else
-      pushAsyncIds(asyncId, triggerAsyncId);
-  }
```

Comment:
History lesson: This was introduced originally because of a crazy method to keep track of the id stack. Which was a lot faster than it is currently, but added an obscene amount of complexity. I hope to simplify that approach in the future and reintroduce it, but seems I overlooked these calls can be removed for the time being.